src/hyperdeck/hyperdeck-core.js

```diff
@@ -206,7 +232,12 @@ function HyperdeckCore(ip) {
     handleConnectionResponse();
   });
   client.setEncoding('utf8');
-  client.on('error', function(e) {
+
+  // https://nodejs.org/api/net.html#net_socket_settimeout_timeout_callback
+  client.setTimeout(cfg.timeout);
```

Comment:
I don't think this makes sense? The documentation says this does nothing except trigger the 'timeout' event which we are not handling

Reply:
Yes, will remove this.